README.md

```diff
@@ -40,9 +40,108 @@ This is a [Devise](https://github.com/plataformatec/devise) extension to add [Tw
 
 ## Pre-requisites
 
-To use the Authy API you will need a Twilio Account, [sign up for a free Twilio account here](https://www.twilio.com/try-twilio).
 
-Create an [Authy Application in the Twilio console](https://www.twilio.com/console/authy/applications) and take note of the API key.
+To use the Twilio Verify API you will need a Twilio Account, [sign up for a free Twilio account here](https://www.twilio.com/try-twilio).
+
+Create an [Twilio Verify Application in the Twilio console](https://www.twilio.com/console/authy/applications) and take note of the API key.
+
+## Getting started
+
+First get your Twilio Verify API key from [the Twilio console](https://www.twilio.com/console/authy/applications). We recommend you store your API key as an environment variable.
+
+```bash
+$ export TWILIO_AUTH_TOKEN=YOUR_TWILIO_AUTH_TOKEN
+$ export TWILIO_ACCOUNT_SID=YOUR_TWILIO_ACCOUNT_SID
+$ export TWILIO_VERIFY_SERVICE_SID=YOUR_TWILIO_VERIFY_SERVICE_SID
+```
+
+Next add the gem to your Gemfile:
+
+```ruby
+gem 'devise'
+gem 'devise-twilio-verify'
```

Comment:
Please correct the name of the gem. Additionally, we haven't uploaded to rubygems.org, so we need to use the git parameter for the gem requirement. I recommend the following:
 ```suggestion
gem 'devise-authy', git: 'https://github.com/MidwestAccessCoalition/twilio-verify-devise.git'
```